src/components/TodoItem.tsx

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+
+import { useState } from 'react';
+import { Todo } from '../types/Todo';
+import classNames from 'classnames';
+import { UpdateTodo } from '../types/Updates';
+
+interface Props {
+  todo: Todo;
+  onDelete: (todoId: number) => void;
+  updateTodo: ({ id, newData }: UpdateTodo) => void;
+  todoLoadingStates: { [key: number]: boolean };
+}
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  onDelete,
+  updateTodo,
+  todoLoadingStates,
+}) => {
+  const { id, completed, title } = todo;
+  const [editing, setEditing] = useState<boolean>(false);
```

Comment:
```suggestion
  const [isEditing, setIsEditing] = useState(false);
```
boolean values should start with `is has`